CONTRIBUTING.md

```diff
@@ -245,3 +246,60 @@ However, these options can lengthen your testing cycle and make the process of c
 ```{bash}
 ./test.sh
 ```
+
+## Releasing to CRAN <a name="cran"></a>
+
+Once substantial time has passed or significant changes have been made to `pkgnet`, a new release should be pushed to [CRAN](https://cran.r-project.org). This is a manual proces, with the following steps.
```

Comment:
`process`

Reply:
good catch. In the most recent commit, I ran `aspell check CONTRIBUTING.md` and found a handful of other spelling issues, all now fixed.